js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-setup-sections.js

```diff
@@ -111,13 +117,20 @@ export default function PaidAdsSetupSections( {
 		amount: paidAds.amount,
 	};
 
+	const formOpts = {
+		budget,
+		budgetMin: 0.3,
```

Comment:
@dsawardekar I think we don't need to pass this value around and instead be a constant defined in `js/src/components/paid-ads/validateCampaign.js`